android/app/src/main/java/io/locha/p2p/service/ChatServiceModule.java

```diff
@@ -302,6 +308,12 @@ public void run() {
                 peerId = intent.getStringExtra("peerId");
                 assert peerId != null;
                 mPromise.resolve(peerId);
+
+                if(!Utils.isConnected(context)){
+                    event.isConnected(Utils.isConnected(context));
+                    return;
+                }
+                event.isConnected(Utils.isConnected(context));
```

Comment:
The event _may_ be notified two times, is this right? Could be simplified to a single line if it's not:

```suggestion
                event.isConnected(Utils.isConnected(context));
```

Reply:
you right xD,  it will be deleted